vmm/src/config.rs

```diff
@@ -176,6 +176,10 @@ pub enum ValidationError {
     DuplicateDevicePath(String),
     /// Provided MTU is lower than what the VIRTIO specification expects
     InvalidMtu(u16),
+    /// PCI segment is reused across NUMA nodes
+    PCISegmentReused(u32, u32, u32),
```

Comment:
Other errors in this enum have Pci rather than PCI.

Reply:
Updated new errors to match convention.